WebMap/static/js/map.js

```diff
@@ -129,6 +140,27 @@ $(document).ready(function () {
             addUserMarker(bMarkerLayer, ".add-b-marker");
         });
 
+        $("#addMarkerBtn").click(function () {
+            showAlert('Place marker on the map', 'info');
+            addUserMarkerRadius(userMarkerLayer, "#addMarkerBtn");
```

Comment:
I think you are using wrong layer. Please use layer that you created:
```suggestion
            addUserMarkerRadius(routeMarkerLayerRadius, "#addMarkerBtn");
```
Use this in all other pleaces where this layer is used

Reply:
done